<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardScreen</class>
 <widget class="QMainWindow" name="DashboardScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1665</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>831</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCommandLinkButton" name="commandLinkButton_4">
       <property name="text">
        <string>All Books</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="text">
        <string>Books By Genre</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCommandLinkButton" name="commandLinkButton_2">
       <property name="text">
        <string>Book Borrow Portal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCommandLinkButton" name="commandLinkButton_5">
       <property name="text">
        <string>Search Student</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCommandLinkButton" name="commandLinkButton_3">
       <property name="text">
        <string>Student Registration Portal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCommandLinkButton" name="commandLinkButton_6">
       <property name="text">
        <string>Book Entry Portal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
